Firestore/Source/Remote/FSTSerializerBeta.mm

```diff
@@ -262,9 +262,7 @@ - (FSTFieldValue *)decodedFieldValue:(GCFSValue *)valueProto {
 
     case GCFSValue_ValueType_OneOfCase_TimestampValue: {
       Timestamp value = [self decodedTimestamp:valueProto.timestampValue];
-      return [FSTTimestampValue
-          timestampValue:[FIRTimestamp timestampWithSeconds:value.seconds()
-                                                nanoseconds:value.nanoseconds()]];
+      return [FSTTimestampValue timestampValue:std::move(value)];
```

Comment:
Optional: consider not moving `Timestamp`, since it's essentially a POD. Feel free to push back if you feel using `move` for consistency is more important.

Reply:
OK. I've changed the parameters to const references too so that we're not tempted to do this.